internal/mempool/reactor_test.go

```diff
@@ -130,6 +130,7 @@ func (rts *reactorTestSuite) waitForTxns(t *testing.T, txs []types.Tx, ids ...ty
 	// ensure that the transactions get fully broadcast to the
 	// rest of the network
 	wg := &sync.WaitGroup{}
+	var count int
```

Comment:
Are we using this for something?

Reply:
whoops. I was doing more logging in this earlier and forgot to remove. all better.